pebble_manager.py

```diff
@@ -18,12 +18,14 @@ def __init__(self, qemu):
         self.handle_stop = None
         self.blobdb = None
         self.watch_version_info = None
+        self.watch_fw_version = None
 
     def connect(self):
         self.register_endpoints()
         self.pebble.connect_via_qemu(self.qemu)
         self.pebble.emu_bluetooth_connection(True)
         self.watch_version_info = self.pebble.get_versions()
+        self.watch_fw_version = self.pebble.get_watch_fw_version()
```

Comment:
This causes the watch to be contacted twice for the same piece of information; it calls pebble.get_versions() internally (libpebble is bad at caching things).